operations/operations/zuul-ci-cd-quickstart-user-guide.md

```diff
@@ -0,0 +1,203 @@
+# Zuul users guide
+
+## Prerequisites
+
+1. Repository is known by your zuul instance
+2. Basic ansible knowledge
+3. Basic yaml knowledge
+4. zuul-client installed
+
+## Who is it for?
+
+You may have heard about zuul and may ask yourself if it is capable to support you.
+Basically everything you use ansible for can be done using zuul. That is not always
+a good thing since you may get careless and your workload will exceed the CI/CD concept.
+
+If you find yourself doing things under the following list you are at the right place.
+
+1. Code testing
+2. Deployment tests using IaC
+
+If you want to, let's say, monitor something using zuul, that is possible but not the
+intended use case.
+
+## Where do I start?
+
+Right in your projects repository! The only prerequisite you need to fulfill, is that
```

Comment:
What is the expected directory structue? Which files are needed? Below there is talk about Projects and so on, but I don't know where these config snippets go.

Reply:
May you recheck?

I've updated some information to address these points.
There is a directory structure added and the SCS Zuul is directly linked.

Reply:
That helped. Thank you!